ArduCopter/tuning.cpp

```diff
@@ -212,5 +212,17 @@ void Copter::tuning() {
      case TUNING_RATE_YAW_FILT:
          attitude_control->get_rate_yaw_pid().filt_hz(tuning_value);
          break;
-    }
+
+     case TUNING_WINCH: {
+         float desired_rate = 0.0f;
+         if (v > 0.8f) {
+             desired_rate = g2.winch.get_rate_max();
+         }
+         if (v < 0.2f) {
+             desired_rate = -g2.winch.get_rate_max();
+         }
+         g2.winch.set_desired_rate(desired_rate);
```

Comment:
Isn't this supposed to be a linear change? Unless I'm missing something this is doing the same as the switch.